MultiServer.py

```diff
@@ -1793,6 +1793,11 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
         elif cmd == 'LocationScouts':
             locs = []
             create_as_hint: int = int(args.get("create_as_hint", 0))
+            client_player = client.slot
```

Comment:
it almost feels wrong to type this in python but i mean why not 🤷 
```suggestion
            client_player = desired_player = client.slot
```
alternatively since args is a dict you could do `desired_player = int(args.get("player", client_player))`

Reply:
Ahhh yes, that "alternative" is what I was looking for but couldn't figure out

I think I tried sth like
```
client_player = int(args.get("player"))
```
but int(None) produces an error. Can't believe I didn't think of just using client_player as the default arg to prevent it